frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -99,6 +99,16 @@ export class TransformTopologyData {
     },
   };
 
+  private transformPods = (pods): K8sResourceKind[] => {
+    return _.map(pods, (pod) =>
+      _.merge(_.pick(pod, 'metadata', 'status', 'spec.containers'), {
+        id: _.get(pod, 'metadata.uid'),
+        name: _.get(pod, 'metadata.name'),
```

Comment:
unless we expect `pod` to be undefined, `_.get` isn't needed here. `metadata.uid` and `metadata.name` are required for an existing pod